cmd/vcluster/cmd/start.go

```diff
@@ -124,6 +124,8 @@ func NewStartCommand() *cobra.Command {
 
 	cmd.Flags().StringVar(&options.DefaultImageRegistry, "default-image-registry", "", "This address will be prepended to all deployed system images by vcluster")
 
+	cmd.Flags().StringVar(&options.PodSecurityStandard, "enforce-pod-security-standard", "privileged", "This can be set to privileged, baseline, restricted (defaults to privileged) and vcluster would make sure during translation that these policies are enforced.")
```

Comment:
it should be possible to set `baseline,restricted` at the same time as those standards are additive (compare pod security standard docs)